public/docs/dart/latest/guide/template-syntax.jade

```diff
@@ -1,407 +1,125 @@
-include ../_util-fns
+extends ../../../ts/latest/guide/template-syntax.jade
 
-+includeShared('{ts}', 'intro')
+block includes
+  include ../_util-fns
+  - var _JavaScript = 'Dart';
+  - var __chaining_op = '<code>;</code>';
+  - var __new_op = '<code>new</code> or <code>const</code>';
+  - var mapApiRef = 'https://api.dartlang.org/stable/1.16.0/dart-core/Map-class.html';
```

Comment:
I guess this is what we need for now, but once https://github.com/dart-lang/api.dartlang.org/issues/62 is fixed let's update this link!